storage/connectors/redis/src/main/java/feast/storage/connectors/redis/writer/RedisFeatureSink.java

```diff
@@ -113,11 +114,29 @@ public RedisFeatureSink withSpecsView(
   @Override
   public PTransform<PCollection<FeatureRow>, WriteResult> writer() {
     if (getRedisClusterConfig() != null) {
-      return new RedisCustomIO.Write(
-          new RedisClusterIngestionClient(getRedisClusterConfig()), getSpecsView());
+      RedisCustomIO.Write writer =
+          new RedisCustomIO.Write(
+              new RedisClusterIngestionClient(getRedisClusterConfig()), getSpecsView());
+
+      if (getRedisClusterConfig().getFlushFrequencySeconds() > 0) {
+        writer =
+            writer.withFlushFrequency(
+                Duration.standardSeconds(getRedisClusterConfig().getFlushFrequencySeconds()));
```

Comment:
Should there be a default here or an exception if it isnt set? In which case would you not have a flush frequency used?

Reply:
Default is specified inside RedisCustomIO https://github.com/feast-dev/feast/pull/866/files#diff-f780c1a978d6503861ed53a1084d79d9R50
Do you want to pull it here?

Reply:
I dont think it needs to be pulled into this location, I just want to make sure there are no silent exceptions. If this value always needs to be set then we need an exception if the predicate fails.

Reply:
I pulled default values to RedisFeatureSink, so all configuration would happen in one place

Reply:
So we have default values & we won't fail in case some config wasn't set